contrib/opencensus-ext-azure/opencensus/ext/azure/events_exporter/__init__.py

```diff
@@ -0,0 +1,24 @@
+from opencensus.ext.azure.common import utils
+from opencensus.ext.azure.common.protocol import Data, Envelope, Event
+from opencensus.ext.azure.log_exporter import AzureLogHandler
+
+
+class AzureEventHandler(AzureLogHandler):
+    def __init__(self, **options):
+        super(AzureEventHandler, self).__init__(**options)
+
+    def log_record_to_envelope(self, record):
+        envelope = Envelope(
+            iKey=self.options.instrumentation_key,
+            tags=dict(utils.azure_monitor_context),
+            time=utils.timestamp_to_iso_str(record.created),
+        )
+
+        envelope.name = "Microsoft.ApplicationInsights.Event"
```

Comment:
I thought we decided to use Logs as the actual telemetry type? This would be `Message` data.

Reply:
But wouldn't that log the data as "searchable text" in the logs table (https://github.com/microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/MessageData.bond#L6)? 

The event definition is different (and this link might be outdated) and more resonant to what we are trying to accomplish here (custom events): https://github.com/microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/EventData.bond#L5

Maybe I'm missing something, please let me know.